build/tf-wrapper.sh

```diff
@@ -133,7 +134,11 @@ tf_validate() {
   else
     if [ -d "$path" ]; then
       cd "$path" || exit
-      terraform show -json "${tmp_plan}/${tf_component}-${tf_env}.tfplan" > "${tf_env}.json" || exit 32
+      if [[ "$runner_env" == "GITHUB" ]]; then
+        terraform-bin show -json "${tmp_plan}/${tf_component}-${tf_env}.tfplan" > "${tf_env}.json" || exit 32
```

Comment:
Could you add a short comment why this is needed?

Reply:
context for the change added